src/TableV8/styled.tsx

```diff
@@ -154,7 +160,7 @@ export const TableHeader = styled(TableHeaderComp, {
   ${(props) =>
     props.canSort &&
     `
-      padding: 0 !important; // padding messes up box-shadow
```

Comment:
removed `!important` in CSS. only needed it before because I was using CSS classnames